invision-app/app/store/actions.js

```diff
@@ -0,0 +1,60 @@
+import {
+    FETCH_MOVIES,
+    CREATE_MOVIE,
+    DELETE_MOVIE,
+    UPDATE_MOVIE,   
+    SORT_BY_RELEASE_DATE,
+    SORT_BY_TITLE,
+    FILTER_BY_GENRE,
+    SHOW_DESCRIPTION_MOVIE
+  } from './types';
+  
+  export function getMovies(){
+      return async(dispatch)=>{        
+          let resp=await fetch('http://localhost:8081/movies');
+          let moviesJson=await resp.json();      
+          dispatch({type:FETCH_MOVIES,payload:moviesJson});
+      };
+  }
+
+  export function createMovie(movie){
+      return{
+          type:CREATE_MOVIE,
+          payload:movie
```

Comment:
One of the best practice is using object as a payload and add to this object needed information using meaningful names. It should help to understand better how the handler, action, reducer should work. For example, DELETE_MOVIE action should create the following action object:
```
{
  type:DELETE_MOVIE,
  payload: {
    id
  }
}
```